docs/core_docs/src/theme/EmbeddingTabs.js

```diff
@@ -0,0 +1,131 @@
+/* eslint-disable react/jsx-props-no-spreading, react/destructuring-assignment */
+import React from "react";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import CodeBlock from "@theme-original/CodeBlock";
+import Npm2Yarn from "@theme/Npm2Yarn";
+
+function InstallationInfo({ children }) {
+  return <Npm2Yarn>{children}</Npm2Yarn>;
+}
```

Comment:
Whats the point of wrapping this and not just calling `Npm2Yarn` inside the component?

Reply:
Thanks, updated this. I had copied `InstallationInfo` from chat model tabs and then deleted some stuff.